src/test/cli/rbd/help.t

```diff
@@ -1665,7 +1665,7 @@
   rbd help mirror image enable
   usage: rbd mirror image enable [--pool <pool>] [--namespace <namespace>] 
                                  [--image <image>] 
-                                 <image-spec> <mode> 
+                                 <image-spec> [<mode>] 
```

Comment:
Just thinking. We made `<mode>` optional only because we did not want to break cli compatibility with pre-snapshot-mode versions. So may be we don't want to "document"/"advertise" that it is optional?

It is like with deprecated commands: we don't remove them but make invisible in documentations. But I am fine with the current version too.

Reply:
I think the optionality is already advertised because the long help text says `mirror image mode (journal or snapshot) [default: journal]`.  `[<mode>]` just makes the short help consistent with the long help.

Reply:
Ok. As I said, I am fine with this.